components/AnswerModal/View.js

```diff
@@ -0,0 +1,93 @@
+import { Dialog, Transition } from '@headlessui/react'
+import { Fragment, useRef } from 'react'
+
+export default function AnswerModalView({ isOpen, closeModal, handleAnswerChange, handleSubmit }) {
```

Comment:
If we have time we can make a Modal that will act as a wrapper around any component (children). That way you don't need to have AnswerModal & QuestionModal. You would just pass in the Answer, Question, or whatever component. Happy to pair on that later.